test/unit/org/apache/cassandra/transport/ClientResourceLimitsTest.java

```diff
@@ -215,9 +215,9 @@ private void backPressureTest(Runnable limitLifter, Consumer<ClientResourceLimit
             // When the client attempts to execute the second query, the backpressure
             // mechanism should report the client connection is paused
             assertTrue(started.await(5, TimeUnit.SECONDS));
-            spinAssertEquals("Timed out after waiting 5 seconds for paused " +
+            spinAssertEquals("Timed out after waiting 10 seconds for paused " +
                              "connections metric to increment due to backpressure",
-                             before + 1, pausedConnections::getValue, 5, TimeUnit.SECONDS);
+                             true, () -> pausedConnections.getValue() >= before + 1, 10, TimeUnit.SECONDS);
```

Comment:
I think we can leave the stricter check in place if we just avoid doing driver initialization.

See https://github.com/apache/cassandra/pull/1045/files#diff-c801dfb00bc39ce5bf66a5876779350dba35def185588d13f5e492b6d79212f7R77 and https://github.com/apache/cassandra/pull/1045/files#diff-822707a68576bd683b72fde39e19ab8a78ed5a1816d95f5d8a00b3f46ac78d03R419

Reply:
+1